### ISSUE TITLE
[Settings] Missing error announcement

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#7]
## Issue summary

When the Settings form is submitted with invalid data, the error messages displayed are not programmatically conveyed to assistive technology users. Focus is reset to the top of the DOM and no notification is provided when the `input` receives focus.

<img width="724" alt="Screen Shot 2020-05-19 at 11 03 12 AM" src="https://user-images.githubusercontent.com/1392632/82343081-64ac3780-99c0-11ea-8251-bca4e635b9a9.png">

#### HTML

```html
<input
  class="text-field__input"
  autocomplete="new-password"
  type="password"
  name="user[password]"
  id="user_password"
/>
<!-- … -->
<div class="text-field__error-message">
  You must enter a password
</div>
```

## Steps to reproduce

1. Go to https://portal.covidshield.app/users with with any supported browser
2. With an active screen reader, submit the Settings form with invalid data

### Expected

- Error messages will be programmatically conveyed to assistive technology users.

### Actual

- Error messages are not announced on focus.

## Recommendation

1. Add a unique `id` to each error message container.
2. On error, add the `aria-describedby` attribute to the `input`, setting its value to the error container `id`.
3. On error, provide extra context by adding the `aria-invalid` attribute on the `input`, setting its value to `true`.
4. Shift keyboard focus to the first `input` in an error state.

#### HTML

```html
<input
  class="text-field__input"
  autocomplete="new-password"
  type="password"
  name="user[password]"
  id="user_password"
  aria-describedby="error-0"
  aria-invalid="true"
/>
<!-- … -->
<div class="text-field__error-message" id="error-0">
  You must enter a password
</div>
```

## Specifications

- **Component:** Settings
- **WCAG Principle:** Understandable
- **WCAG SC:** 3.3.1 Error Identification
- **Severity:** High
- **Effort:** Medium
